src/components/totalPrice/index.tsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import { ShoppingProps } from "../../utils/interfaces";
+import './styles.css'
+
+const TotalPrice = ({shoppingList}: ShoppingProps): JSX.Element => {
+    let totalPrice = 0
+    shoppingList.map((purchase) => {
+        if (purchase.price) {
+            totalPrice = totalPrice + purchase.price
+        }
+        return totalPrice;
+    })
```

Comment:
This is incorrect. Map should be used to return a new array, yet you doesn't assign it to anything. For this kind of tasks a special method in JS arrays exist - reduce, use it.